### ISSUE TITLE
Parse Error : syntax error on line 1 > 1 | <?php PHP_VERSION

### ISSUE THREAD

**Issue by zeyadsharo** [shufo/vscode-blade-formatter#64]
## Template

Parse Error : syntax error on line 1 > 1 | <?php PHP_VERSION

## Context (Environment)

Parse Error : syntax error on line 1 > 1 | <?php PHP_VERSION

- write your environment

php 7.4 
laravel 8

## Description

![image](https://user-images.githubusercontent.com/33178745/107856625-77ae4400-6e3a-11eb-9d16-1f5563f3415a.png)

**Comment by betomedin** [comment]
php 5.6
Laravel 4.2

**Comment by brada1703** [comment]
I'm having the same issue with PHP 8.0.2 and Laravel 8.12

**Comment by furopi** [comment]
I'm getting this error, too. Please help...
php 8.0.2
Laravel 8

**Comment by shufo** [comment]
Seems

```php
<?php PHP_VERSION
```

is not right PHP syntax because PHP_VERSION is constant.

```php
❯ php
<?php PHP_VERSION

Parse error: syntax error, unexpected end of file in Standard input code on line 2
```

Below syntax is ok

```php
❯ php
<?php
echo PHP_VERSION;
7.4.8
```

Can you provide any example template?

**Comment by furopi** [comment]
It happens with the default Laravel start page.... The sample page that is provided after a fresh Laravel 8 installation.

**Comment by shufo** [comment]
It seems parse error while formatting `{{ PHP_VERSION }}` as php string.

To workaround this, you can use `phpversion()` function as equivalent code on `welcome.blade.php`

```blade
<div class="ml-4 text-center text-sm text-gray-500 sm:text-right sm:ml-0">
    Laravel v{{ Illuminate\Foundation\Application::VERSION }} (PHP v{{ phpversion() }})
</div>
```

CLI test

```bash
❯ echo "{{ phpversion() }}" | ./bin/blade-formatter --stdin
{{ phpversion() }}

```

**Comment by furopi** [comment]
Ok, I can live with that. But honestly I think this problem should be fixed for other users who tend to not read the docs and will abandon the plugin or think it's not working... Just my two cents...

On the other hand I must say you did a very decent job with this plugin, using it on a daily basis! Taihen yoku gambarimashita, Shuhei-san. Doumo arigatou gozaimasu! Greetings from Germany :beer: to Japan :japan:, Florian

**Comment by shufo** [comment]
Just released [v0.7.3](https://github.com/shufo/vscode-blade-formatter/compare/v0.7.2...v0.7.3) with this issue fixed. 
Thanks for reporting issue @zeyadsharo @furopi @brada1703 @betomedin !
